estate_auction/models/estate_property.py

```diff
@@ -0,0 +1,93 @@
+from odoo import api, fields, models
+from odoo.exceptions import UserError
+
+class EstateProperty(models.Model):
+    _inherit = 'estate.property'
+
+    property_sale_format = fields.Selection(
+        string="Sell Type",
+        selection=[
+            ('auction', "Auction"),
+            ('regular', "Regular"),
+        ],
+        default='regular',
+        copy=False
+    )
+    auction_stage = fields.Selection(
+        string="Auction Stage",
+        selection=[
+            ('template', "Template"),
+            ('auction', "Auction"),
+            ('offer_accepted', "Offer Accepted"),
+        ],
+        default='template',
+        copy=False
+    )
+    auction_end_time = fields.Datetime(string="Auction End Time")
+    highest_bidder = fields.Many2one(string="Highest Bidder",
+        comodel_name='res.partner',
+        compute='_compute_highest_bidder',
+        store=True
+    )
+    invoice_ids = fields.One2many(string="Invoice", comodel_name="account.move", inverse_name="estate_property_id")
+    invoice_count = fields.Integer(string="Invoice Count", compute="_compute_invoice_count")
+
+    def write(self, vals):
+        if self.auction_stage == '03_offer_accepted' and self.state == 'sold':
```

Comment:
```suggestion
        if self.auction_stage == 'offer_accepted' and self.state == 'sold':
```
don't you think these should be change too

Reply:
yes that was my mistake will change it.